workitem/workitem.go

```diff
@@ -123,13 +136,26 @@ func (wi WorkItem) ChangeSet(older convert.ChangeDetector) ([]convert.Change, er
 		})
 		return changes, nil
 	}
-	thisCopy := make([]string, len(bcThis))
-	olderCopy := make([]string, len(bcOlder))
-	copy(thisCopy, bcThis)
-	copy(olderCopy, bcOlder)
-	sort.Strings(thisCopy)
-	sort.Strings(olderCopy)
-	if !reflect.DeepEqual(thisCopy, olderCopy) {
+	// because go has a fundamentally broken handling of interface{}
+	// types, we need to do a totally overshooting manual conversion
+	// here.
+	thisCopyStr := make([]string, len(bcThis))
+	for i := range bcThis {
+		thisCopyStr[i], ok = bcThis[i].(string)
+		if !ok {
+			return nil, errors.New("Boardcolumn slice values are not of type string")
```

Comment:
In fact, board columns will soon be of type `[]uuid.UUID` and not of type `[]string` ;)